src/physics/thermal_solver_rework.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
```

Comment:
I kept them separate for the sake of comparing outputs, but you're right that it would be more helpful to actually just apply the changes to the thermal solver to better understand the code changes!